release.sh

```diff
@@ -62,5 +94,9 @@ fi
 # only file which is changing
 git commit -m "Version $VERSION" $commitFiles
 git tag "$VERSION"
-git push github.com:taskcluster/taskcluster-client.py master "$VERSION"
-python setup.py sdist upload
+git push "${OFFICIAL_GIT_REPO}" "$VERSION:$VERSION"
```

Comment:
we still need to push the `master` reference or else we'll never have the changes from this scripts commits to setup.py and readme.md on the master branch of the repo.